client/app/dedicatedCloud/datacenter/datastore/dedicatedCloud-datacenter-datastore.html

```diff
@@ -2,107 +2,56 @@
-                        <td data-ng-bind="datastore.size.value + ' ' + ('unit_size_' + datastore.size.unit | translate)"></td>
-                        <td>
-                            <span class="d-block"
-                                  data-translate="{{ 'dedicatedCloud_tab_datastores_billing_' + datastore.billing }}">
-                            </span>
-                            <button type="button"
-                                    class="btn btn-link p-0"
-                                    data-ng-show="datacenter.model.canOrderFilers && datastore.billing == 'HOURLY'"
-                                    data-ng-click="setAction('resource/upgrade/dedicatedCloud-resource-upgrade', {
-                                        upgradeType: datacenter.model.isUpgradable && 'demoToMonthly' || 'hourlyToMonthly',
-                                        type: 'filer',
-                                        id: datastore.id
-                                    })"
-                                    data-translate="dedicatedCloud_updateResource_toMonthly">
-                            </button>
-                        </td>
-                        <td data-translate="{{ 'dedicatedCloud_tab_datastores_status_' + datastore.status }}"></td>
-                        <td>
-                            <span data-ng-if="datastore.billing === 'HOURLY' && isNan(datastore.consumption)"
-                                  data-translate="dedicatedCloud_tab_datastores_hourly_consumption_na">
-                            </span>
-                            <span data-ng-if="datastore.billing === 'HOURLY' && datastore.consumption <= 1"
-                                  data-translate="dedicatedCloud_tab_datastores_hourly_consumption_1"
-                                  data-uib-tooltip="{{ 'dedicatedCloud_tab_host_hourly_usage_update' | translate:{ t0: (datastore.consumptionLastUpdate|date:'short') } }}"
-                                  data-tooltip-placement="top">
-                            </span>
-                            <span data-ng-if="datastore.billing === 'HOURLY' && datastore.consumption > 1"
-                                  data-ng-bind="'dedicatedCloud_tab_datastores_hourly_consumption_other' | translate:{ t0: datastore.consumption }"
-                                  data-uib-tooltip="{{ 'dedicatedCloud_tab_host_hourly_usage_update' | translate:{ t0: (datastore.consumptionLastUpdate|date:'short') } }}"
-                                  data-tooltip-placement="top">
-                            </span>
-                        </td>
-                    </tr>
-                </tbody>
-            </table>
-        </div>
-        <div data-pagination-server-side
-             data-pagination-server-side-function="loadDatastores"
-             data-pagination-server-side-paginated-stuff="datastores"
-             data-pagination-server-side-table-loading="loading">
-        </div>
-    </div>
+    <button type="button"
```

Comment:
You replace `table` with `datagrid` but not `button` with `oui-button` ?

Reply:
`<extra-top>` from https://master.ui-kit.ovh/#!/oui-angular/datagrid instead you mean? :)